src/components/EmailSubscription/EmailSubscription.js

```diff
@@ -27,9 +27,8 @@ const styles = {
     border: '2px solid #FFF',
     color: '#FFF',
     fontFamily: 'Kievit-Book',
-    fontSize: '14px',
+    fontSize: '0.875em',
```

Comment:
If you want to get fancier, this `'0.875em'` value can go in a variable.

Reply:
not sure I want to get fancier at this point in the game... sprint is over for me today.